web-bundle/src/main/java/com/graphhopper/resources/SPTResource.java

```diff
@@ -218,7 +225,24 @@ public Response doGet(
                     } catch (IOException ex) {
                         throw new RuntimeException(ex);
                     }
-                });
+                };
+
+                if (fullyExpandPillarNodes) {
+                    PillarEdgeResolver pillarEdgeResolver = new PillarEdgeResolver(l -> {
+                        IsoLabelWithCoordinates label = isoLabelWithCoordinates(nodeAccess, l.original_label);
+
+                        // TODO: make sure node/adj_node is correct for coord/prevCoord
+                        label.coordinate = l.node;
+                        label.prevCoordinate = l.prev_node;
+                        isoLabelConsumer.accept(label);
+                    }, graph);
+                    shortestPathTree.search(snap.getClosestNode(), pillarEdgeResolver);
+                } else {
+                    shortestPathTree.search(snap.getClosestNode(), l -> {
+                        IsoLabelWithCoordinates label = isoLabelWithCoordinates(nodeAccess, l);
+                        isoLabelConsumer.accept(label);
```

Comment:
Was wondering will this path ever run - if not could a comment be added to clarify?

Reply:
added an optional parameter in the URL params that allow either branch to run